models/src/main/scala/coop/rchain/models/rholang/implicits.scala

```diff
@@ -316,6 +320,8 @@ object implicits {
         case EAndBody(EAnd(p1, p2))     => p1.get.connectiveUsed || p2.get.connectiveUsed
         case EOrBody(EOr(p1, p2))       => p1.get.connectiveUsed || p2.get.connectiveUsed
         case EMethodBody(e)             => e.connectiveUsed
+        case EMatchesBody(EMatches(process, pattern)) =>
+          process.get.connectiveUsed || pattern.get.connectiveUsed
```

Comment:
You shouldn't have to call `.get` anymore. There're probably couple residues. There must be an implicit that goes from `Par => Option[Par]` that makes compiler accept those constructs.

Reply:
done